### PR TITLE
Update asset loader

### PR DESCRIPTION
**Pull request by guybedford**
Updates to release:
* https://github.com/zeit/webpack-asset-relocator-loader/releases/tag/0.5.2.
* https://github.com/zeit/webpack-asset-relocator-loader/releases/tag/0.5.3

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1251,7 +1251,7 @@
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz#d291c6a4e97989b5c61d9acf396ae4fe133a718d"
   integrity sha512-NuHqBY1PB/D8xU6s/thBgOAiAP7HOYDQ32+BFZILJ8ivkUkAHQnWfn6WhL79Owj1qmUnoN/YPhktdIoucipkAQ==
 
-"@zeit/webpack-asset-relocator-loader@0.5.1":
+"@zeit/webpack-asset-relocator-loader@0.5.2":
   version "0.5.1"
   resolved "https://registry.yarnpkg.com/@zeit/webpack-asset-relocator-loader/-/webpack-asset-relocator-loader-0.5.1.tgz#796e454c18fefa1b08be322c7908e8fa2294ea3e"
```
Comment (styfle) [inline yarn.lock]: This is the wrong sha, right?

Comment (guybedford) [reply yarn.lock]: Weird, when I did `yarn install` it acted like this was fine.

Comment (styfle) [reply yarn.lock]: Try `yarn add `@zeit/webpack-asset-relocator-loader@0.5.2`

**Comment by styfle** [comment]
Does this close any issues?

**Pull request merged**